include/openssl/ossl_typ.h

```diff
@@ -1,196 +1,16 @@
 /*
- * Copyright 2001-2018 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 2019 The OpenSSL Project Authors. All Rights Reserved.
  *
- * Licensed under the OpenSSL license (the "License").  You may not use
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
```

Comment:
No, do not change the license in 1.1.1

Reply:
That change was not intentional of course.